Start on Database restore

AndreRl opened the pull request:
Considering adding a note regarding IcyBoards.

For #153 

Image links will need altering as they currently use Gyazo.

AndreRl committed:
Start on Database restore

AndreRl committed:
MySQL Database Restore

Completed doc for MySQL. Image sizes will need reviewing and linked from MyBB site instead of Gyazo.

AndreRl committed:
Update database-restore.md

AndreRl committed:
Update database-restore.md

AndreRl committed:
Update database-restore.md

AndreRl committed:
PgSQL and SQLite PhpLiteAdmin Content

PostgreSQL SSH instructions finished as well as SQLite PhpLiteAdmin.

AndreRl committed:
PhpPgAdmin

PostgreSQL content added.

AndreRl commented:
@euantorano Are you able to review when you have a minute? Aware of the Gyazo images, unsure of how to address them!

euantorano commented:
Yep, I can have a look tonight after work. If you don't hear anything from me, give me a nudge on Discord later @AndreRl.

euantorano commented on 1.8/install/database-restore.md:
```diff
@@ -0,0 +1,143 @@
+- /Database-Restore.html
+---
+
+To restore a database to your forum, you will first need to know what database type your forum is using in order to determine which instructions to follow. MyBB supports the following database types:
```
`database type` might make sense to be worded as `database engine`, as I believe that's what we call it in the installer (I think, off the top of my head).

euantorano commented on 1.8/install/database-restore.md:
````diff
@@ -0,0 +1,143 @@
+
+1. After logging into your server via SSH, run the following:
+
+``` mysql -u YOUR_MYBB_DB_USER -p YOUR_MYBB_DBNAME < /PATH/TO/DUMP.SQL ``` 
````
This should be written as:

    ```
    mysql -u YOUR_MYBB_DB_USER -p YOUR_MYBB_DBNAME < /PATH/TO/DUMP.SQL
    ```

Note the difference in formatting when viewed.

euantorano reviewed (changes requested):
Regarding images, the files should be added here: https://github.com/mybb/docs.mybb.com/tree/gh-pages/assets/images/1.8

I would probably create a subfolder in there, then add the images. For an example of using an image from here, see the following page: https://github.com/mybb/docs.mybb.com/blob/797eee50acf98f75df916262ceb0fd8a8111c3c4/1.8/faq/using-a-cdn-with-mybb.md

AndreRl committed:
Images upload

AndreRl committed:
Update database-restore.md

Images uploaded

AndreRl commented:
Cheers @euantorano. Changes have been made.

AndreRl committed:
Update database-restore.md

Indentation fixed

euantorano commented on 1.8/install/database-restore.md:
```diff
@@ -0,0 +1,163 @@
+- /Database-Restore.html
+---
+
+To restore a database to your forum, you will first need to know what database engine your forum is using in order to determine which instructions to follow. MyBB supports the following database types:
```
Probably best to change the other `database types` to `database engines` too 😛

euantorano commented on 1.8/install/database-restore.md:
```diff
@@ -0,0 +1,163 @@
+
+To restore a database to your forum, you will first need to know what database engine your forum is using in order to determine which instructions to follow. MyBB supports the following database types:
+
+* MySQL (In this demonstration, PhpMyAdmin will be used)
```
It might also be nice to link to the PhpMyAdmin/PhpLiteAdmin/PhpPgAdmin project homepages here I think.

euantorano commented on 1.8/install/database-restore.md:
```diff
@@ -0,0 +1,163 @@
+
+## Optional
+
+1. Now that you have restored your database, we are not quite done yet. You may now need to configure your database settings (.../inc/config.php).
```
Some extra details on what might need to be changed here would be nice, along with a comment showing the interesting parts of the file (eg: the database configuration). Would be nice to quote the path, like `./inc/config.php` too (and notice the single `.`, rather than `...`).

AndreRl replied:
Possibly, but as this is only database restoring this page assumes that a forum is already in-place and we're only restoring a database, not creating a new forum with an old database. If this is the case and we're only restoring a database, only the ```table_prefix``` would need to be set as everything else would be set via the original install. Perhaps making this clearer will suffice (running the install, then restoring)? @euantorano

euantorano replied:
True, and for a straight restore you'd probably not need to change the prefix either.

euantorano reviewed (changes requested):
Looks pretty good, just a few more minor comments/changes.

AndreRl committed:
Update database-restore.md

AndreRl committed:
Update database-restore.md

AndreRl committed:
Optional Updated

Upload paths added for IcyBoards Restore. 
Added info on database configuration in ./inc/config.php

euantorano commented:
Anybody else want to give this a quick look over for typos I may have missed or anything @mybb/community-team?

AndreRl commented:
@Ben-MyBB Can you take a look when you get a moment?

AndreRl commented:
@euantorano Can we merge this in the next day or two if no other input is contributed?

euantorano commented:
Sure, will do. 

> On 31 Jul 2018, at 04:10, Andre R. | Wires <notifications@github.com> wrote:
> 
> @euantorano Can we merge this in the next day or two if no other input is contributed?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

AndreRl commented:
Nudge @euantorano :)
